cmake/External/aotriton.cmake

```diff
@@ -16,6 +16,33 @@ if(NOT __AOTRITON_INCLUDED)
   set(__AOTRITON_EXTERN_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/aotriton")
   set(__AOTRITON_INSTALL_DIR "${PROJECT_SOURCE_DIR}/torch")
   add_library(__caffe2_aotriton INTERFACE)
+
+  # AOTriton package information from GitHub Release Pages
+  # Replaces .ci/docker/aotriton_version.txt
+  # Note packages information may have versions skipped (due to no ABI breaks)
+  # But they must be listed from lower version to higher version
+  set(__AOTRITON_VER "0.8b")
+  set(__AOTRITON_MANYLINUX_LIST
```

Comment:
Please use the exact `__AOTRITON_MANYLINUX_LIST` `__AOTRITON_ROCM_LIST` `__AOTRITON_CI_COMMIT` and `__AOTRITON_SHA256_LIST`  in PyTorch main branch.
These lists describe the packages available at https://github.com/ROCm/aotriton/releases/tag/0.8b
and for now we are not shipping manylinux_2_17 anymore.